balancer/frontend-v2
# Refactor chain configs. Consolidate into one location

# Description

This PR is the beginning of moving all chain configuration files into one location so when we add a new chain we only have to worry about one folder instead of references all over the app. It does the following:

- Switch all our config files from .json to .ts so we can have variables and comments and more flexibility with them. 
- Configuring for each chain is now one folder instead of one file so that the config can be split into general/contracts/pools etc and be easier to parse. 
- Move all pool allowlist/blocklist config from one big constants files into a pools config file for each chain. 

There is no functionality changes in this PR, everything should look and work exactly the same as production.

## Type of change


## How should this be tested?

- Ensure everything functions exactly the same as current develop. 
- Allowlisted pools should be viewable, blocked pools should not. 
- All networks should connect and load information correctly. 

## Checklist:
